src/application/reportLists/reportListGetResult.ts

```diff
@@ -0,0 +1,13 @@
+import { ReportListData } from './reportListData'
+
+/**
+ * レポートリスト取得結果
+ */
+export class ReportListGetResult {
+  /**
+   * コンストラクタ
+   *
+   * @param reportListData レポートリストデータ
+   */
+  public constructor(public readonly reportListData: ReportListData) {}
+}
```

Comment:
```suggestion
export class ReportListGetResult {
  /**
   * コンストラクタ
   *
   * @param reportListData レポートリストデータ
   */
  public constructor(public readonly reportListData: ReportListData) {}
}
```

指摘ではないんですけど、せっかくここまで記述したコードでreportListが取得できてるので、そのことを先生がmanakanを実際に操作して確認できる部分があったほうがいいですかね。

Reply:
認識が合っていて良かったです！！

Reply:
中鉢先生に、実際にプロダクトを触ってみてもらって、提出物が取得できるのを確認してもらった方がレビューしやすいかなと思ったんですよね。
なので、少しだけ#102 のPRで削除したコミットの型定義を直して、残しておいてもよかったかなって思ったんですよ

Reply:
そうですねローディングはいらないです。コース名やレポートタイトルを今回のレポートリストのデータにするのはどうですか？？